src/components/QuickGrid/rowGrouper.ts

```diff
@@ -1,11 +1,13 @@
 import * as _ from 'lodash';
-import { GroupRow, IGroupBy } from './QuickGrid.Props';
+import { GroupRow, IGroupBy, GridColumn } from './QuickGrid.Props';
 class RowGrouper {
     groupByColumns: Array<IGroupBy>;
+    columns: Array<GridColumn>;
     expandedRows: any;
-        constructor(groupByColumns, expandedRows) {
+        constructor(groupByColumns, expandedRows, columns) {
         this.groupByColumns = groupByColumns.slice(0);
```

Comment:
We raised the TypeScript version, so we can use spread operator instead of slice, [...groupByColumns]

Reply:
Replaced